src/toTransform.js

```diff
@@ -35,14 +35,50 @@
     return 'scale3d(' + valuesArray.join(', ') + ')';
   }
 
+  function convertPath (properties) {
+    var offsetPath = null;
+    if ('offset-path' in properties) {
+      offsetPath = internalScope.offsetPathParse(properties['offset-path']);
+    }
+
+    if (offsetPath === undefined || offsetPath === null || offsetPath.type !== 'path') {
+      return null;
+    }
+
+    var offsetDistance;
+    if (('offset-distance' in properties) && (properties['offset-distance'] !== undefined)) {
```

Comment:
I expect the parse function to return undefined if these conditions are true, you shouldn't need to check it here.